docs/matlab.rst

```diff
@@ -33,21 +27,17 @@ We will show:
 
    - To install the OMERO.matlab toolbox https://www.openmicroscopy.org/omero/downloads/
 
-   - The Image Processing toolbox is only necessary for the images
-      analysis. This is a convenient toolbox for analysis purpose.
-      You do not need to install that toolbox to integrate OMERO and
-      MATLAB.
+   - The Image Processing toolbox is only necessary for the image analysis. This is a convenient toolbox for analysis purpose. You do not need to install that toolbox to integrate OMERO and MATLAB.
```

Comment:
you will need it for the segmentation part

Reply:
In that case, it is not clear to me where did I install it. 
My Matlab works fine with the whole workflow, getting the segmentation and the ROIs back to OMERO. But I am not aware of installing any Image Processing toolbox. This is not clear to me... @dominikl

Reply:
Then you probably have it already installed. Check your preferences?
![Screen Shot 2020-03-02 at 09 35 34](https://user-images.githubusercontent.com/6575139/75663829-668c1780-5c69-11ea-8de5-7633336167c0.png)

Reply:
But yes, maybe we could add a link to matlab, to make it clearer what this is about:
https://uk.mathworks.com/products/image.html

Reply:
@dominikl yes, indeed, as checked in Preferences, I have it already installed.

Did Matlab install it for me on the strenght of my account history ? I re-installed Matlab on this machine just couple of months ago. Yes, I will have a look about adding that link @dominikl is pointing to.

Reply:
I guess so. It probably checks if it's part of your license and then installs it automatically. Unfortunately I can't remember how I installed it in first place.